spec/hardware/system.fmf

```diff
@@ -20,6 +20,21 @@ description: |
            # Integer or string, required number of NUMA nodes.
            numa-nodes: 2|">= 2"
 
+           # System management interface
+           management-interface:
+
+               # String, a name of the management interface protocol.
+               protocol: IPMI
+
+               # Integer or string, an ID or name of the management interface vendor.
+               vendor: "~ MontaVista Software, Inc."
```

Comment:
We're already using pairs like `vendor` and `vendor-name` or `model` and `model-name`. Shall we possibly apply the same consistent approach here as well?

Reply:
Same for `product`, that's `device` and `device-name`.

Reply:
so, will it be `model`(`-name`) or `device`(`-name`) ? I don't see the difference between the two. Is the specification already inconsistent?

Reply:
We are using `device` and `vendor` for ID-like properties, and `device-name` and `vendor-name` for label-like properties, names and such.

`model` and `model-name` are defined only for CPU, and it's true that's pretty much the same meaning. I'd recommend adding `device` and `device-name` properties to CPU group rather then adding `model` and `model-name` anywhere else.

Reply:
Dammit. No, `device` and `device-name` it is. We shall root this out. I'm not going to support seven different aliases.

Reply:
For disks it is not even an alias, `device-name` is not in the document.

Reply:
So changed to :

* protocol
* device and device-name
* vendor and vendor-name